test/jdk/sun/security/krb5/MicroTime.java

```diff
@@ -44,9 +44,9 @@ public static void main(String[] args) throws Exception {
                 count++;
             }
         }
-        // We believe a nice KerberosTime can at least tell the
-        // difference of 100 musec.
-        if (count < 10000) {
+        // Before this change, KerberosTime was implemented in milliseconds.
+        // Now there should be more.
+        if (count < 1001) {
```

Comment:
Can you make this comment more specific? What change was this? And what is the precision before and after the change?

Reply:
Updated. I was pretending the comment was written right when the bug was fixed.